src/main/java/com/celements/collections/service/StreamScriptService.java

```diff
@@ -29,12 +29,16 @@
     return nullGuard(stream).collect(Collectors.toCollection(() -> collection));
   }
 
-  public String join(Stream<String> stream, String separator) {
```

Comment:
Can you add a "toString" for each element? Comming from velocity generics are not garantied.

Reply:
this comment references the old signature. please check the new one.